app/models/form/question.rb

```diff
@@ -58,6 +59,10 @@ def check_answer_label
     @check_answer_label ||= I18n.t("forms.#{form.start_date.year}.#{copy_key}.check_answer_label", default: "")
   end
 
+  def check_answer_prompt
+    @check_answer_prompt || I18n.t("forms.#{form.start_date.year}.#{copy_key}.check_answer_prompt", default: nil).presence || generate_check_answer_prompt
+  end
```

Comment:
Added this property to the question model and translation files. If a value is set in the question, it will use that. Otherwise, it will use any value set in the translation file. Setting `default: nil` in the `I18n.t` method call ensures that if the translation is not found it will return `nil` instead of returning a fallback string. This ensures if neither is available, it will generate a default check answer prompt.